packages/app/src/app/pages/Sandbox/Editor/Header/ForkButton/ForkButton.tsx

```diff
@@ -106,98 +106,116 @@ interface ForkButtonProps {
 export const ForkButton: React.FC<ForkButtonProps> = props => {
   const { state } = useOvermind();
   const { user } = props;
+  let teams: TeamItem[] = [];
+  let currentSpace: TeamItem = null;
+  let otherWorkspaces: TeamItem[] = [];
 
   const userSpace = state.dashboard.teams.find(
     t => t.id === state.personalWorkspaceId
   )!;
 
-  const allTeams: TeamItem[] = [
+  const allTeams: {
+    id: string;
+    name: string;
+    avatarUrl: string;
+    userAuthorizations: MemberAuthorization[];
+  }[] = [
     userSpace,
     ...state.dashboard.teams.filter(t => t.id !== state.personalWorkspaceId),
-  ].map(team => ({
-    teamId: team.id,
-    teamName: team.name,
-    teamAvatar: team.avatarUrl,
-    userAuthorizations: team.userAuthorizations,
-  }));
+  ];
```

Comment:
Since `userSpace` can be null (I think?)

```suggestion
  ].filter(Boolean);
```